File: planning-idsdp-rms-ti-backend/src/main/java/com/ibcs/idsdp/trainninginstitute/web/dto/request/ProgressVerificationRequest.java
package com.ibcs.idsdp.trainninginstitute.web.dto.request;

import com.ibcs.idsdp.common.model.domain.MinioAttachment;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ProgressVerificationRequest {

    private Long trainingInstituteProfileId;

    private String examinerUserId;

    private LocalDate verificationDate;

//    private String researchTitle;

    private Integer numberOfManPower;

    private String organizationActivity;

    private String monitoring;

    private String comment;

    private MinioAttachment nothi;

    private Long fiscalYearId;

    private Long proposalId;
}

File: planning-idsdp-rms-frontend/src/app/main/modules/training-institute/models/progress-verification.request.ts
import { FileUploadModel } from "../../../shared/model/file-upload.model";
import { ProposalModel } from "./proposal.model";

export class ProgressVerificationRequest {


    comment: string;
    examinerUserId: string;
    fiscalYearId: number;
    monitoring: string;
    nothi: FileUploadModel;
    numberOfManPower: number;
    organizationActivity: string;
    trainingInstituteProfileId: number;
    verificationDate: Date;
    proposalId: number;
    proposalModel: ProposalModel;
}
